execute/exectypes/observation.go

```diff
@@ -32,17 +96,26 @@ type Observation struct {
 	// execute report.
 	Messages MessageObservations `json:"messages"`
 
+	// TokenData are determined during the second phase of execute.
+	// It contains the token data for the messages identified in the same stage as Messages
+	TokenData TokenDataObservations `json:"tokenDataObservations"`
+
 	// Nonces are determined during the third phase of execute.
 	// It contains the nonces of senders who are being considered for the final report.
 	Nonces NonceObservations `json:"nonces"`
 }
 
 // NewObservation constructs a Observation object.
```

Comment:
typo `an Observation object`

Reply:
lol, haven't touched that, but I can fix :P